.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           node-version: ${{ matrix.node }}
 
       - name: Install deps and build (with cache)
-        run: npm ci
+        run: yarn install
```

Comment:
This looks like it might be a mistake? i'm not seeing a yarn lock file so should this still be npm ci?

Reply:
The yarn lock file is generated only in the github workflow. 

For some reason, this works with yarn, but when I install with npm and then try to run `rollup` commands via the `build:umd` scripts, I get rollup can't be found errors: https://github.com/apollographql/embeddable-explorer/actions/runs/4672936172/jobs/8275624073

I haven't been able to dive deep into it yet

Reply:
I switched all the run commands to npm in the above link, with no success.  I see what you are saying though I'll hack at it more this morning 🙏🏻

Reply:
bizarre!  It's almost like the node env is prod for some reason and it's skipping installing dev deps 🤔

Reply:
Okay it was just an npm version mismatch, we weren't installing the right npm version